tests/functional/VestingEscrow/test_delegate.py

```diff
@@ -3,17 +3,20 @@
 
 def test_delegate(deployed_vesting, recipient, voting_adapter):
     data = voting_adapter.encode_delegate_calldata(recipient)
```

Comment:
Consider adding a negative test with invalid encoded data.

Reply:
Also consider adding test for undelegation (delegation to zero address)

Reply:
Addressed in https://github.com/lidofinance/lido-vesting-escrow/pull/9/commits/4246b8ddac3e00a53972baffef78ab722a00212f.